projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -289,6 +289,21 @@ protected static function get_default_tables() {
 			),
 			'links'                      => $wpdb->links, // TODO describe in the array format or add exceptions.
 			'options'                    => $wpdb->options, // TODO describe in the array format or add exceptions.
+			'woocommerce_products'       => array(
+				'table'                     => $wpdb->posts,
```

Comment:
Since each field in the `wc_product_meta_lookup` table can be updated independently, it doesn't provide a reliable basis for checksum calculation. If we use that table, we would need to include multiple or even all fields in the checksum, which is not ideal.

Instead, I use the `post_modified_gmt` field from the posts table for this purpose.

While WPCOM may not always have a posts table, our cache table includes a `date_modified_gmt` field, which serves as an equivalent for checksum comparison if I’m not mistaken.

Reply:
I like this solution, which should keep the query and checksums as lean as possible. 
In my tests, things worked mostly as expected, adding, updating and deleting new products. All changes were picked up :)